brightchain-lib/src/lib/interfaces/network/node.ts

```
@@ -0,0 +1,160 @@
+  PEER_CONNECT = 'node:peer:connect',
+  PEER_DISCONNECT = 'node:peer:disconnect',
+  BLOCK_TEMPERATURE = 'node:block:temperature',
+  BLOCK_LOCATION = 'node:block:location',
+  SYNC_START = 'node:sync:start',
+  SYNC_COMPLETE = 'node:sync:complete',
+}
+
+/**
+ * Node event
+ */
+export interface NodeEvent<T = any> {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)